### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by penwelldlamini** [matt8707/hass-config#40] [Q&A] [answered]
Hi I love your design but can you please help on the installation. I tried looking for this design on HACS and could not find it. How can I install this this design. Im using a raspberry Pi3

**Comment by Ripper2909** [comment]
There is no HACS integration. You have to modify your code by yourself looking at his. 😉

**Comment by matt8707** [comment, marked as answer]
https://github.com/matt8707/hass-config/blob/master/INSTALL.md

**Reply by tismofied** [reply]
I love love your dashboard.
Any chance adding this dashboard to HACS?

**Reply by CumpsD** [reply]
Thanks for your explanation! I just discovered your theme on the HA forum, looked at the code, and was left wondering (I only got HA for a week :p)

This will be my starting point, thanks :)

**Reply by suppqt** [reply]
Many thanks ! Works fine so far, one more question? Which files are needed to implement the glassy effect including the background blur?
cant find it yet

**Reply by matt8707** [reply]
https://caniuse.com/css-backdrop-filter

https://github.com/matt8707/hass-config/blob/a14d5447a908b5f2098bad94dc8017b27466e9bb/include/themes.yaml#L354-L358

**Reply by moelfus** [reply]
Thanks for sharing your awesome theme! I have just a quick question where do I copy the yaml files. I'm using hassio and suppose the have to go to config/www/community bcause I cannot find a hacsfiles folder.

**Reply by matt8707** [reply]
https://hacs.xyz/docs/categories/plugins

**Reply by grimmaldus** [reply]
If I may ask I'm having problem with the background.
It loads ok if I use your theme in managed ui, however if I switch to ui-lovelace.yaml it doesn't want to load at all.

**Comment by Crazy38TR** [comment]
Hello, I did the install.md steps one by one, but it gave an error. What should I do?

![Ekran görüntüsü 2022-10-13 220039](https://user-images.githubusercontent.com/57983995/195684447-4b50429b-b8e2-415f-a359-ebc3c5936736.png)
